automan/api/projects/annotations/annotation_manager.py

```diff
@@ -120,6 +120,30 @@ def delete_annotations(self, dataset_id, storage):
         for annotation in annotations:
             self.delete_annotation(annotation.id, storage)
 
+    def get_active_frame(
+            self, project_id, user_id, annotation_id,
+            frame, order_flag):
```

Comment:
`order_flag` of boolean type is not clear. you should use declaration argument name.